uk.co.compucorp.civicrm.hrcore/hrcore.php

```diff
@@ -26,7 +26,10 @@ function hrcore_civicrm_config(&$config) {
  * @param array $tasks
  */
 function hrcore_civicrm_searchTasks($objectName, &$tasks) {
-  if ($objectName !== 'contact' || !CRM_Core_Permission::check('create users')) {
+  $isContact = $objectName === 'contact';
```

Comment:
I would agree that this logic belongs somewhere else. I don't really like the method of adding a lot of functions inside hrcore.php though, and prefer to use classes. For this case it might be overkill though. 

I'll change the logic to remove the early return and add the search tasks inside an if statement.